packages/assets-controllers/CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- `TokenListController` now persists `tokensChainsCache` via `StorageService` using per-chain files to reduce write amplification ([#7413](https://github.com/MetaMask/core/pull/7413))
```

Comment:
Nice!!

Should we mark this as breaking since we are adding a new action?

Reply:
should this be a breaking change ?